core/app/models/spree/order.rb

```diff
@@ -884,12 +881,11 @@ def after_cancel
       payments.completed.each { |payment| payment.cancel! unless payment.fully_refunded? }
       payments.store_credits.pending.each(&:void_transaction!)
 
-      send_cancel_email
```

Comment:
I ran into a situation recently where the removal of this method caused problems with an extension that tried to override it. Would it be possible to leave the `send_foo_email` methods in place and have them call the publish method instead? That would allow extension authors to update their tricks before things stop working.